### DISCUSSION TITLE
remove wind down and change wind up mod name

### DISCUSSION THREAD

**Discussion by GDcheerios** [ppy/osu#18861] [Ideas]
they both have the exact same range in mod customization.

I feel like there is no point of having two mods that do the same thing but just different initial configs. At that point it's more a preset.

**Comment by LeNitrous** [comment]
The same can be said for Grow/Deflate as these mods were created prior to mod customization. Even more so with Easy, Hard Rock, Nightcore, Double Time, Daycore, Half Time. As much as they sound like presets, it somewhat adds a more complexity to mod customization for the user's end to achieve the same effects as what the original mods did whereas the current iteration is clear for what its intents are to the end user.

Maybe at some point where mod presets are a thing it would make more sense. For now I feel like it would be too early.

**Reply by GDcheerios** [reply]
yeah, this makes sense. I made a PR(#18873) prior to this discussion and ppy told me the importance of discussion before making a PR. He also basically summarized what you said to me.
